src/Private/Authentication.ps1

```diff
@@ -2133,6 +2098,28 @@ function Find-PodeAuth {
     return $PodeContext.Server.Authentications.Methods[$Name]
 }
 
+
+function Expand-PodeAuthMerge {
+    param (
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string[]]
+        $Names
+    )
+    $authNames = @{}
+    foreach ($authName in  $Names) {
+        $_auth = $PodeContext.Server.Authentications.Methods[ $authName]
+        if ( $_auth.merged) {
+            foreach ($a in $_auth.Authentications) {
+                $authNames[$a] = ''
+            }
+        } else {
```

Comment:
Merged authentications can contain further merged authentications, so you'll need to do a recursive call on `Expand-PodeAuthMerge` here.